expr/funcs.go

```diff
@@ -2,55 +2,16 @@ package expr
 
 import "github.com/raintank/metrictank/api/models"
 
-//go:generate stringer -type=argType
-type argType uint8
-
-// argument types. to let functions describe their inputs and outputs
-// potentially not as strict as reality (e.g. movingAverage windowsize is categorized as a str) that's why we have the extra validation step
-const (
-	series      argType = iota // a single series
-	seriesList                 // a list of series
-	seriesLists                // one or multiple seriesLists
-	integer                    // number without decimals
-	integers                   // one or multiple numbers without decimals
-	float                      // number potentially with decimals
-	str                        // string
-	boolean                    // True or False
-)
-
-type optArg struct {
-	key string
-	val argType
-}
-
 type Func interface {
```

Comment:
my plan is to move and rename github.com/raintank/metrictank/expr to github.com/grafanalabs/graphiteng or something (or perhaps github.com/graphite-ng/process or something) in the near future.